plugins/flatpak/gs-flatpak.c

```diff
@@ -3566,18 +3553,16 @@ gs_flatpak_app_copy (GsFlatpak *self,
 							     cancellable,
 							     cancelled_id);
 		g_child_watch_add (child_pid, copy_process_watch_cb, helper);
-	} else if (error && *error) {
+	} else
```

Comment:
So, the thing about not having curly braces on one-liners is that if it's an if/else block where the counterpart has curly braces you should do it too... (Sorry if this is too nitpicky but I think this is how it usually goes upstream)